iphone/Maps/UI/Search/TableView/MWMSearchCommonCell.mm

```diff
@@ -22,7 +22,18 @@ - (void)configureWith:(SearchResult * _Nonnull)result isPartialMatching:(BOOL)is
   [super configureWith:result isPartialMatching:isPartialMatching];
   self.locationLabel.text = result.addressText;
   [self.locationLabel sizeToFit];
+
+  // Define emoji font with fallback
+  UIFont * emojiFont = [UIFont fontWithName:@"OrganicMapsEmojiFont" size:13];
```

Comment:
Should the size be taken from the font used in the cell instead of hard-coding it?